src/main/webapp/script.js

```diff
@@ -19,13 +19,18 @@ google.charts.load('current', {
 });
 
 function passQuery() {
+  document.getElementById('map-title').innerHTML = '';
   document.getElementById('result').style.display = 'block';
   const information = document.getElementById('more-info');
   information.innerText = 'Please wait. Loading...';
   const query = new FormData(document.getElementById('query-form'));
   const personType = query.get('person-type');
   const action = query.get('action');
   const location = query.get('location');
+  const region =
+      location === 'state' ? 'U.S. state' : 'State Name county';
+  const title = 'Population who ' + action +
+      ' in each ' + region + ' (' + personType + ')';
```

Comment:
For increased readability, maybe do a find and replace for converting dashes to spaces when making the title? (so that eg all-ages appears as all ages)

Reply:
Done!